syntax = "proto3";

message player_enter_gas_response {
	uint64 player_id = 1;
	uint32 result = 2;
}